### PR TITLE
Update GH Action Workflow

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
## Proposed changes

Learning from past mistakes ❌

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -3,13 +3,13 @@ name: Build and Deploy
 on:
   push:
     branches:
-      - master
+      - main
```
Comment (cainlevy) [inline .github/workflows/deploy.yml:6]: so merging `origin/dev` to `origin/main` triggers a release to staging? 👍

**Review by cainlevy — APPROVED** [review]

**Comment by cainlevy** [comment]
we may want to add the deployment process to the readme

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (gabrielsaruhashi) [reply .github/workflows/deploy.yml:6]: yep!

Comment (gabrielsaruhashi) [reply .github/workflows/deploy.yml:6]: oh wait, dev --> would be production

anything to --> dev would be a staging release

**Pull request merged**